packages/shared/src/components/buttons/common.ts

```diff
@@ -209,8 +208,6 @@ export const useGetIconWithSize = (
       className: classNames(
         icon.props.className,
         !iconOnly && '!h-6 !w-6 text-base',
-        !iconOnly && iconPosition === ButtonIconPosition.Left && '-ml-2 mr-1',
-        !iconOnly && iconPosition === ButtonIconPosition.Right && '-mr-2 ml-1',
```

Comment:
Why did we end up removing this? This might affect any existing implementations. If there is none, then we can move forward.